test/integration/SimplePublishConsume.test.ts

```diff
@@ -192,4 +192,77 @@ describe('Simple Publish & consume test', async () => {
     }
     */
   })
+  it('should publish and transfer an NFT', async () => {
+    const nft = new Nft(web3)
+    const Factory = new NftFactory(addresses.ERC721Factory, web3)
+    const accounts = await web3.eth.getAccounts()
+    const publisherAccount = accounts[0]
+    const newOwnerAccount = accounts[1].toLowerCase()
+
+    const nftParams: NftCreateData = {
+      name: 'testNFT',
+      symbol: 'TST',
+      templateIndex: 1,
+      tokenURI: '',
+      transferable: true,
+      owner: publisherAccount
+    }
+    const erc20Params: Erc20CreateParams = {
+      templateIndex: 1,
+      cap: '100000',
+      feeAmount: '0',
+      paymentCollector: '0x0000000000000000000000000000000000000000',
+      feeToken: '0x0000000000000000000000000000000000000000',
+      minter: publisherAccount,
+      mpFeeAddress: '0x0000000000000000000000000000000000000000'
+    }
+    const result = await Factory.createNftWithErc20(
+      publisherAccount,
+      nftParams,
+      erc20Params
+    )
+    const erc721Address = result.events.NFTCreated.returnValues[0]
+    const datatokenAddress = result.events.TokenCreated.returnValues[0]
+
```

Comment:
can we add a subgraph query here (same as below), to make sure we get the right owner after create and before transfer ?

Reply:
Sure